tutorials/chapter2.md

```diff
@@ -443,4 +378,4 @@ The subscriber factory in the ZMQ case is always the ```kpsr::zmq_mdlw::FromZmqM
 <a name="zmq-env"></a>
 ### ZMQ Env
 
-The ```kpsr::zmq_mdlw::ZMQEnv``` is similar to the ```kpsr::dds_mdlw::DDSEnv``` in that it is a wrapper to the ```kpsr::YamlEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as DDS or ROS.
+```kpsr::zmq_mdlw::ZMQEnv``` is a wrapper to the ```kpsr::YamlEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as ROS.
```

Comment:
```suggestion
```kpsr::zmq_mdlw::ZMQEnv``` is a wrapper to the JSON based ```kpsr::ConfigurationEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as ROS.
```
This is unrelated to DDS, but a necessary fix.

Reply:
Done.